crates/cli/src/commands/setup.rs

```diff
@@ -154,38 +136,42 @@ fn already_setup_message(installed_bin_path: PathBuf) {
 }
 
 fn finished_message(
-    installed_bin_path: PathBuf,
-    updated_profile_path: Option<PathBuf>,
+    proto: &ProtoResource,
     exported_content: Option<String>,
+    path_was_updated: bool,
 ) {
-    if let Some(profile_path) = updated_profile_path {
-        println!(
-            "Successfully installed proto to {} and updated profile {}!",
```

Comment:
Letting users know which profile was updated is very important for users, since they don't always choose it.

Reply:
Yeah, it was just moved to https://github.com/moonrepo/proto/pull/456/files#diff-a31902d06ac6afdddf64feb76dc589f59e39a33434c352de60ba84f8beca4427R116 instead

Reply:
Moved it back to finished_message